src/static/js/modules/jquery/cf_notifier.js

```diff
@@ -0,0 +1,169 @@
+/* ==========================================================================
+   jquery.cf_notifier
+   ========================================================================== */
+
+'use strict';
+
+var $ = require( 'jquery' );
+var handlebars = require( 'handlebars' );
+
+function init() {
+  var notifier = {
```

Comment:
Objects like this that are defining a lot of functions should probably be their own modules.

Here and throughout.